### PR TITLE
Dt edits

### PR DESCRIPTION
**Pull request by dethanga**
added a new file with instructions about deleting old instances of Docker container and images.

### CODE REVIEW

**Comment by dethanga** [comment]
Hi Anne, 

I have added a new file (5.md) - To remove an existing Docker container and images.
I also updated the JSON file.

Please review and approve.

File: labs/docker-101/5.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# How to remove an existing Docker Container and Images
+
+In this section, you will learn how to remove/delete old instances of Docker container and its images. Also, you can learn and practice additional Docker commands.
+
+1. In the Windows PowerShell mode, run the following command to get the list of containers:
```
Comment (annegentle) [inline labs/docker-101/5.md]: "In Windows PowerShell or a terminal window, run the following command to get the list of containers:"

File: labs/docker-101/5.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+   
+   `docker ps -a`
+   
+   ![](assets/images/Docker-new1.png)  
+    
+2. If you want to remove the old instances of the container, execute the following command:
+   
+   **Note**: Docker will not allow to remove the running container directly, hence you need to apply `docker stop`
+    command to stop the process.
```
Comment (annegentle) [inline labs/docker-101/5.md]: "Docker will not allow you to remove" < insert "you"

Or, "Docker prevents removal of a running container, so you need to use the `docker stop` command to stop the process before removing the instance."

File: labs/docker-101/5.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+2. If you want to remove the old instances of the container, execute the following command:
+   
+   **Note**: Docker will not allow to remove the running container directly, hence you need to apply `docker stop`
+    command to stop the process.   
+    
+    `docker rm $(docker ps -qa)`
+   
+   ![](assets/images/Docker-new2.png)  
+   
+3. To get the list of existing docker images, execute the following command:
```
Comment (annegentle) [inline labs/docker-101/5.md]: docker > Docker

File: labs/docker-101/5.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+|docker ps|This command is used to list the running containers.|
+|docker ps -a|This command is used to show all the running and exited containers.| 
+|docker exec -it <container id> bash|This command is used to access the running container.|
+|docker stop <container id>|This command stops a running container.|
+|docker commit <conatainer id> <username/imagename>|This command creates a new image of an edited container on the local system.|
+|docker push <username/image name>|This command is used to push an image to the docker hub repository.|
+|docker images|This command lists all the locally stored docker images.|
+|docker rm <container id>|This command is used to delete a stopped container.|
+|docker rmi <image-id>|This command is used to delete an image from local storage.|
+|docker build <path to docker file>|This command is used to build an image from a specified docker file.|
```
Comment (annegentle) [inline labs/docker-101/5.md:48]: This table is wonderful! Nice work.

**Review by annegentle — CHANGES_REQUESTED** [review]
Minor suggestions inline - this is a great addition!

**Pull request merged**